tools/ecfr-parser/main.go

```diff
@@ -136,6 +136,10 @@ func run() error {
 
```

Comment:
More spacing issues in this file as well

Reply:
same as above.